src/area-chart/__tests__/area-chart-initial-state.test.tsx

```diff
@@ -8,7 +8,6 @@ import { KeyCode } from '@cloudscape-design/test-utils-core/dist/utils';
 import popoverStyles from '../../../lib/components/popover/styles.css.js';
 import { warnOnce } from '../../../lib/components/internal/logging';
 import { cloneDeep } from 'lodash';
```

Comment:
Missing `import '../../__a11y__/to-validate-a11y';`

I'd prefer to add it to every file that requires the extra function for better visibility or only add it once in jest unit setup.

Reply:
or not using a matcher, because is it really that useful?

What was wrong with `await expectNoAxeViolations(container);`?

Reply:
The original function, when failed, pointed to the lines within itself where the expect statements were located, which is not very useful. It was also limiting the possibilities of formatting the error description.